etc/kayobe/environments/ci-multinode/apt.yml

```diff
@@ -0,0 +1,4 @@
+---
+# Disable pulling from pulp repositories, as they currently do not have Ubuntu Jammy packages.
+apt_repositories: "{{ [] if ansible_facts['distribution_release'] != 'jammy' }}"
+apt_disable_sources_list: "{{ false if ansible_facts['distribution_release'] != 'jammy' }}"
```

Comment:
`apt_disable_sources_list: "{{ ansible_facts['distribution_release'] != 'jammy' }}"`

Reply:
Dont want to change it to true if its been set false somewhere else. Only want to set to false or leave it as it is.

Reply:
I'm open to being proven wrong, but I don't think Ansible variables work like that. My understanding is that if the condition is false and you don't provide an else, you get `None`.

Reply:
Ah I see